api-report/container-runtime.api.md

```diff
@@ -241,6 +241,15 @@ export class FluidDataStoreRegistry implements IFluidDataStoreRegistry {
     get IFluidDataStoreRegistry(): this;
     }
 
+// @public (undocumented)
+export function getDeviceSpec(): {
```

Comment:
one small feedback. it would be good if we didn't need to export this function from the package. essentially, we should put in a file, and export it from the file, but not export that file in index.ts. that will allow usage within the package, but not outside of it.

Reply:
another option is to change the export of containerRuntime.ts in index.ts to explicitly export everything but this, similar to the gc file. this might be the easiest way.